src/components/details/_macro.njk

```diff
@@ -39,7 +39,8 @@
                     {{
                         onsButton({
                             "type": "button",
-                            "text": params.button.open or params.button.close,
+                            "text": params.button.close,
+                            "buttonContext": (params.button.context | default(params.title)) + " content",
```

Comment:
Won't be able to hard code text strings into the macro as it will need to be translatable.